src/app/(core)/(home)/commissions.tsx

```
@@ -1,61 +1,136 @@
+                            variants={{
+                                hidden: { opacity: 0, y: 20 },
+                                visible: { opacity: 1, y: 0 }
+                            }}
+                            className="break-inside-avoid"
+                        >
+                            <CommissionCard commission={commission} />
+                        </motion.div>
+                    ))
+                )}
+            </motion.div>
+            <div ref={ref}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)